### ISSUE TITLE
[Visual Requirements-Collect Tab]: Luminosity contrast ratio of selected tab text is less than minimum required ratio 4.5:1

### ISSUE THREAD

**Issue by polatengin** [microsoft/vsts-extension-retrospectives#769]
Note: Same issue reproes throughout the application.

Other instances:
Other instance 1: Collect tab
Path: Retrospective page-->collect tab
Issue: Luminosity contrast ratio of the time text (Ex: 20 December 2023 at 3:07 pm) text present in the tiles is less than 4.5:1, it is "3:1". It should be greater than or equal to the minimum required ratio 4.5:1.
Note: Same issue reproes throughout the application.

Other instance 2: Votes tab
Path: Retrospective page-->Votes tab
Issue: Luminosity contrast ratio of '0' and 'December...' text is less than the minimum required ratio of 4.5:1.
Current ratio of '0': 3.7:1
It should be greater than or equal to the minimum required ratio of 4.5:1
PFA (Other Instance_2_Luminosity contrast ratio of '0' and 'December...' text is less than the minimum required ratio.)

Other instance 3: Team Assessment.
Path: Retrospective page-->Team Assessment.
Issue: Luminosity contrast ratio of Multiple text text is less than the minimum required ratio of 4.5:1.
Multiple text are 'All answers will be saved anonymously' and 1 to 10 text present in the dialog.
Current ratio of 'All answers will be saved anonymously': 3.3:1
Current ratio of 10 text is: 2.3:1.
It should be greater than or equal to the minimum required ratio of 4.5:1
Same issue is observed for the 1 to 9 text present on the colors.

Impact on a Person with Disability (PWD):
Low vision or color sensitive users are facing difficulty as Luminosity contrast ratio of selected tab text is less than minimum required ratio 4.5:1.

Test Environment:
OS: Windows 11 [Version 23H2 (OS Build 22631.2715)]
Browser: Edge Version (119.0.2151.97), Chrome Version (119.0.6045.200)
URL: [Work items - Boards (visualstudio.com)](https://worldwidelearning.visualstudio.com/MCAPS%20Accessibility/_workitems/myactivity/)
Tools: Accessibility insights for web, color contrast analyzer

Repro Steps:
Step 1: Open MCAPS ADO in Edge/Chrome browser.
Step 2: Navigate to 'Retrospective' in left navigation and invoke it.
Step 3: Navigate to the collect tab and invoke it. Run AI.
Step 4: Verify whether luminosity contrast ratio of selected tab text is less than minimum required ratio 4.5:1.

Actual Result: 
Luminosity contrast ratio of selected tab text is less than minimum required ratio 4.5:1, it is 3.8:1

Expected Result:
Luminosity contrast ratio of selected tab text should be greater than or equal to the minimum required ratio 4.5:1.

Suggestive fix
Adjust the foreground color or background color of the selected tab to meet the required contrast ratio 4.5:1.

K4W:
Title: WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (.stage-text)

Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)

Target application: Retrospectives - Boards - https://worldwidelearning.visualstudio.com/MCAPS%20Accessibility/_apps/hub/ms-devlabs.team-retrospectives.home

Element path: .external-content--iframe;.active > .stage-text

Snippet: <p class="stage-text">Collect</p>

How to fix:
Element has insufficient color contrast of 3.79 (foreground color: #0078d4, background color: #ebebeb, font size: 9.0pt (12px), font weight: bold). Expected contrast ratio of 4.5:1

MAS Reference:
[MAS 1.4.3 – Contrast (Minimum).docx (sharepoint.com)](https://microsoft.sharepoint.com/:w:/s/accessibility/ERk6r5xEUrFPgymR3OCLyF4BsjeJjFgpdTc4dJfpsWPbgg?e=nWciye)

WCAG Reference:
[Understanding Success Criterion 1.4.3: Contrast (Minimum) (w3.org)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html)

Reference Link:
https://worldwidelearning.visualstudio.com/MCAPS%20Accessibility/_workitems/edit/1713024#13212667
